src/checkbox.cpp

```diff
@@ -78,7 +78,7 @@ void CheckBox::draw(NVGcontext *ctx) {
     nvgFill(ctx);
 
     if (mChecked) {
-        nvgFontSize(ctx, 1.8 * mSize.y());
+        nvgFontSize(ctx, detail::__icon_scale * mSize.y());
```

Comment:
Third icon scaling change.  I got lazy and used one globally, this one may need to change.

Reply:
I liked the bigger check mark better. Sounds like you'll need an extra factor of ``1.2f``.